src/js/components/Select/SelectContainer.js

```diff
@@ -342,9 +352,35 @@ class SelectContainer extends Component {
     return result;
   };
 
+  renderClear = () => {
+    console.log('in here');
+    const { renderClearValue, clearValueLabel, theme } = this.props;
+    if (renderClearValue) {
+      return renderClearValue({ onClear: this.clearSelection });
+    }
+    return (
+      <Box flex={false}>
+        <Button
+          tabIndex="-1"
+          hoverIndicator="background"
+          onClick={this.clearSelection}
+        >
+          <Box
+            background="light-4"
```

Comment:
Not a must at this point, but we are trying to stay away from the `light-*` colors these days, it is not released officially yet but you can read about it in the *NEW* sections on this wiki https://github.com/grommet/grommet/wiki/Color-Properties

bottom line, it might be better to use `background-back` color?

Reply:
sure thing i ll have a look :)